docs/API/swagger.yaml

```diff
@@ -446,21 +480,12 @@ paths:
         content:
           application/json:
             schema:
-              $ref: '../../schema/system/integration-instance.schema'
+              $ref: '../../schema/system/integration-put.schema'
```

Comment:
why do we need a dedicated resource for the put operation ? it is not a conventional  REST operation